Makefile

```diff
@@ -334,48 +336,10 @@ else
 	sudo -E ./hack/serve.sh
 endif
 
-#------------------------------------------------------------------------------#
-
-# Container Commands
```

Comment:
I thought these were used for general development rather than for CI.  oe.we should still have some of these for building local containers and images to run against.  is that not right?

Reply:
I have no idea. What are the current local use cases people have for the containers?

Reply:
chatted in discord: convenience for managing OCI images for those that want to use that workflow, and as a form of distribution using docker.  i think we only need the busybox/alpine targets for manual testing .. the others can all  go afaict.